### PR TITLE
chore: prepend @aws-amplify scoping pt 3

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes
Dependencies Prepending aws-amplify to :
- cli-shared-interfaces
- amplify-console-hosting
- amplify-container-hosting
- amplify-dotnet template provider.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [n/a] Relevant documentation is changed or added (and PR referenced)
- [n/a] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [x] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by evcodes** [comment]
Changed the branch name because circle ci was not recognizing the changes on the previous branch in fork.

**Comment by evcodes** [comment]
the linting problem was introduced by me in other PRs

File: packages/amplify-cli-shared-interfaces/API.md [hunk 1]
```diff
@@ -1,86 +1,88 @@
-## API Report File for "amplify-cli-shared-interfaces"
+## API Report File for "@aws-amplify/amplify-cli-shared-interfaces"
 
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
+
 // @public (undocumented)
 export interface ICommandInput {
-  // (undocumented)
```
Comment (edwardfoyle) [inline packages/amplify-cli-shared-interfaces/API.md:8]: should this formatting be reverted?

Comment (evcodes) [reply packages/amplify-cli-shared-interfaces/API.md:8]: I didn't change this formatting, just reran the verify api-extract script

**Review by kuhlmanp — APPROVED** [review]

**Review by edwardfoyle — APPROVED** [review]

**Pull request merged**